tests/test_position_sequence.py

```diff
@@ -0,0 +1,147 @@
+import pytest
+
+from useq import GridRelative, MDASequence
+
+
+@pytest.fixture
+def mda() -> MDASequence:
+    return MDASequence(
+        axis_order="tpgcz",
+        stage_positions=[(10, 20, 0), {"name": "test", "x": 10, "y": 20, "z": 50}],
+        channels=[{"config": "Cy5", "exposure": 50}],
+    )
+
+
+def _update_pos_sequence(mda: MDASequence, axis: str):
```

Comment:
i'm not a fan of this pattern... it requires someone reading the tests to jump all over the place.  You only ever use each axis argument ("c", "z", etc...) once in the tests here, so do the replacement in the test.  then it's more obvious to the reader what the input of the test actually was

Reply:
also, you don't *have* to use a fixture for the original mda.  Rather than creating a very small `mda` fixture, that you then selective replace in each test.  Just create the mda you want to test inside of each test